share/ansible_navigator/utils/catalog_collections.py

```diff
@@ -250,7 +244,7 @@ def worker(pending_queue: multiprocessing.Queue, completed_queue: multiprocessin
 
         try:
             (doc, examples, returndocs, metadata) = get_docstring(
```

Comment:
Why doesn't this function accept pathlib objects? Even if we want strings for some reason, I'd move `str(path)` inside of the function.

Reply:
@webknjaz note that this function is from ansible/ansible (`ansible.utils.plugin_docs.get_docstring`).  We could change it there, but it's not just a simple function update in this same PR.